test/presubmit-tests.sh

```diff
@@ -56,4 +56,9 @@ function build_tests() {
   return ${failed}
 }
 
+function pre_integration_tests() {
+  subheader "Building go code"
+  go build -v ./... || return 1
```

Comment:
If the intention here is to build kn, it would be better doing it in e2e-tests.sh so you have a straightforward test procedure.

Reply:
I originally did that and then moved it here just because it let the client build before starting the whole e2e setup phase. But, I was on the fence and happy to move it back.

Reply:
Got it. Then build kn before calling initialize() in e2e-tests.sh.

Reply:
Will do. Thanks for the review!